examples/vag_cdc.c

```diff
@@ -0,0 +1,260 @@
+/*
+ * VAG_CDC.c
+ *
+ * Created: 23.06.2013 20:00:51
+ *  Author: Dennis Schuett, dev.shyd.de
+ */
+
+#define F_CPU 8000000UL
+
+#include <inttypes.h>
+#include <avr/io.h>
+#include <avr/interrupt.h>
+#include <util/delay.h>
+
+#include "uart.h"
+
+#define UART_BAUD_RATE 9600
+#define LED_PWR PA0
+#define RADIO_OUT PD2
+#define FT_CBUS1 PD3
+#define RADIO_OUT_IS_HIGH (PIND & (1<<RADIO_OUT))
+
+#define CDC_PREFIX1 0x53
+#define CDC_PREFIX2 0x2C
+
+#define CDC_END_CMD 0x14
+#define CDC_PLAY 0xE4
+#define CDC_STOP 0x10
+#define CDC_NEXT 0xF8
+#define CDC_PREV 0x78
+#define CDC_SEEK_FWD 0xD8
+#define CDC_SEEK_RWD 0x58
+#define CDC_CD1 0x0C
+#define CDC_CD2 0x8C
+#define CDC_CD3 0x4C
+#define CDC_CD4 0xCC
+#define CDC_CD5 0x2C
+#define CDC_CD6 0xAC
+#define CDC_SCAN 0xA0
+#define CDC_SFL 0x60
+#define CDC_PLAY_NORMAL 0x08
+
+#define MODE_PLAY 0xFF
+#define MODE_SHFFL 0x55
+#define MODE_SCAN 0x00
+
+uint16_t captimehi = 0;
+uint16_t captimelo = 0;
+uint8_t capturingstart = 0;
+uint8_t capturingbytes = 0;
+uint32_t cmd = 0;
+uint8_t cmdbit = 0;
+uint8_t newcmd = 0;
+uint8_t shutdownpending = 0;
+
+uint8_t getCommand(uint32_t cmd);
+void shutdown();
+void softreset();
+
+volatile uint8_t cd;
+volatile uint8_t tr;
+volatile uint8_t mode;
+
+ISR(INT0_vect) //remote signals
+{
+	if(RADIO_OUT_IS_HIGH)
+	{
+		if (capturingstart || capturingbytes)
+		{
+			captimelo = TCNT1;
```

Comment:
[`TCNT` is a timer.](http://www.avrbeginners.net/architecture/timers/timers.html).